### PR TITLE
Deploy Android & iOS artifacts to new OSS repos

### PR DESCRIPTION
**Pull request by matteblair**
This is a simpler approach to https://github.com/tangrams/tangram-es/pull/1783

 - iOS releases are deployed to a Bintray repo: https://bintray.com/tangrams/cocoapods/Tangram-es
 - Android releases are deployed to a Bintray repo: https://bintray.com/tangrams/maven/tangram
 - Android snapshots are deployed to OJO: https://oss.jfrog.org/artifactory/webapp/#/artifacts/browse/simple/General/oss-snapshot-local/com/mapzen/tangram/tangram

TODO:
- [x] Finalize Bintray repository names
- [x] Update release checklist
- [x] Update Android library usage information (resolve from Jcenter now)
- [x] Update iOS podspec with new download location

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,17 +57,18 @@ jobs:
...
-      - run: cd platforms/ios/framework/ && zip ${CIRCLE_ARTIFACTS}/pod.zip README.md
-      # Add the license file.
-      - run: zip ${CIRCLE_ARTIFACTS}/pod.zip LICENSE
-      # Upload the Cocoapods archive to S3.
-      - run: aws s3 cp ${CIRCLE_ARTIFACTS}/pod.zip s3://ios.mapzen.com/tangram-releases/tangram-release-${CIRCLE_TAG}.zip
+      - run: zip -r pod.zip build/ios-framework-universal/Release/TangramMap.framework
+      # Add the readme and license files.
+      - run: zip pod.zip platforms/ios/framework/README.md LICENSE
+      # Upload and publish the Cocoapods archive to Bintray.
+      - run: jfrog bt config --user=${BINTRAY_USER} --key=${BINTRAY_API_KEY} --licenses=MIT
```
Comment (tallytalwar) [inline .circleci/config.yml]: I assume you will be adding `BINTRAY_USER` and `BINTRAY_API_KEY` to circleci environment?

File: platforms/ios/framework/Tangram-es.podspec [hunk 2]
```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.documentation_url = 'https://mapzen.com/documentation/tangram/ios-framework/0.9.0/'
 
   s.source = {
-    :http => "http://ios.mapzen.com/tangram-releases/tangram-release-#{s.version}.zip"
+    :http => "https://dl.bintray.com/tangrams/cocoapods/:#{s.version}-release.zip"
```
Comment (tallytalwar) [inline platforms/ios/framework/Tangram-es.podspec]: We should update the homepage above, to maybe point to github repo url. Similarly the documentation link to github.com/tangrams/tangram-docs.

Actually a few things here:
- author: tangram-es team?, with a shared email address?
- social_media_url:? tangramJS twitter handle?

File: .circleci/config.yml (hunk 1 above)
Comment (tallytalwar) [reply .circleci/config.yml]: Similar for travis.

Comment (tallytalwar) [reply .circleci/config.yml]: Ok Confirmed these environment variables are added to the 2 ci environments.
Will also make sense to clean the environment variables on ci to reflect the ones not being used now. Example MAPZEN_API_KEY or S3 credentials.

**Comment by tallytalwar** [comment]
One more thing we can probably update here is the artifact location on oss.jfrog.org. Currently its in "mapzen/tangram/tangram". Does it make sense to have the artifacts/snapshots live under this path?

I think the path is automatically created? But I could be wrong here.

File: .circleci/config.yml (hunk 1 above)
Comment (matteblair) [reply .circleci/config.yml]: Yeah I will clean up the CI environment variables after this is merged.

**Comment by matteblair** [comment]
@tallytalwar I just pushed some changes to the podspec, take a look now and give a 👍 on the PR if it looks alright. 

To your other question: the artifact location on oss.jfrog.org is determined automatically by our fully-qualified Maven package name, which is `com.mapzen.tangram.tangram`.

File: .travis.yml [hunk 3]
```diff
@@ -52,4 +52,4 @@ script:
 
 after_success:
     - source ./scripts/travis/script_deploy_android_snapshot.sh
-    - source ./scripts/travis/script_deploy_android_demo.sh
+    #- source ./scripts/travis/script_deploy_android_demo.sh
```
Comment (tallytalwar) [inline .travis.yml]: Just a note, if this was intended to be commented, and if so, does it make sense to take it out completely? I am neutral to it. Also if we decide to take the demo deployment out, maybe remove the shell script also.

Comment (matteblair) [reply .travis.yml]: Yeah I did comment this intentionally in a7d94f3

I expect to re-enable this later, but it shouldn't block the other changes.

Comment (tallytalwar) [reply .travis.yml]: Cool, just wanted to clarify.

**Comment by tallytalwar** [comment]
:shipit:

**Pull request merged**